hazelcast/src/test/java/com/hazelcast/map/query/QueryIndexingTest.java

```diff
@@ -134,41 +122,10 @@ private static Config newConfig(final Map<Integer, Employee> employees) {
             } else {
                 val = new Employee(i, "name" + i, "city" + i, 0, true, i);
             }
-            employees.put(i, Mockito.spy(val));
+            Employee spy = MockUtil.serializableSpy(Employee.class, val);
```

Comment:
Could you add a few lines of comment about what is happening here? I'm confused.